helpfiles/batch.txt

```diff
@@ -0,0 +1,20 @@
+
+batch OPERATION ARGUMENT...
+
+This operation expects another operation and its arguments and runs it without
+user interaction. slackroll will error out if user input would be needed to
+select a specific package version among several. In addition, all other
+interactive operations are done automatically with a predefined and documented
```

Comment:
Remove "and documented". It's implicit because the documentation follows.

Reply:
Addressed